tests/meta/observations/test_meta_upload_observations.py

```diff
@@ -146,6 +149,22 @@ def test_is_duplicate_loqusdb_id(
     assert is_duplicate is True
 
 
+def test_check_customer_loqusdb_permissions(
+    customer_rare_diseases: Customer,
+    customer_balsamic: Customer,
+    mip_dna_observations_api: MipDNAObservationsAPI,
+):
+    """Test customers Loqusdb permissions."""
+
+    # GIVEN a MIP observations API, a Rare Disease customer and a Cancer customer
+
+    # WHEN verifying the permissions for Loqusdb upload
+    mip_dna_observations_api.check_customer_loqusdb_permissions(customer_rare_diseases)
```

Comment:
Should you add an assert here for when it goes well?

Reply:
Thanks @henrikstranneheim! 🥇 🦸‍♂️